flutter-for-react-native/index.md

```diff
@@ -0,0 +1,2411 @@
+  on the fly, for example, in hero animations. The following example specifies
+   named routes in the `MaterialApp` widget.
+
+<!-- skip -->
+```Dart
+// Flutter
+class NavigationApp extends StatelessWidget {
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+            ...
+      routes: <String, WidgetBuilder>{
+        '/a': (BuildContext context) => new usualNavscreen(),
+        '/b': (BuildContext context) => new drawerNavscreen(),
+      }
+            ...
+  );
+  }
+}
+```
+
+To navigate to a named route, the [of](https://docs.flutter.io/flutter/widgets/Navigator/of.html) method of the `Navigator` widget is used
+to specify the `BuildContext` (a handle to the location of a widget in the
+widget tree). The name of the route is passed to the `pushNamed` function to
+navigate to the specified route.
+
+<!-- skip -->
+```Dart
+Navigator.of(context).pushNamed('/a');
+```
+
+You can also use the push method of `Navigator` which adds the given [`route`](https://docs.flutter.io/flutter/widgets/Route-class.html)
+to the history of the navigator that most tightly encloses the given
+[`context`](https://docs.flutter.io/flutter/widgets/BuildContext-class.html), and transitions to it. In the following example, the [`MaterialPageRoute`](https://docs.flutter.io/flutter/material/MaterialPageRoute-class.html)
+widget is a modal route that replaces the entire screen with a platform-adaptive
+transition. It takes a [`WidgetBuilder`](https://docs.flutter.io/flutter/widgets/WidgetBuilder.html) as a required parameter.
+
+<!-- skip -->
+```Dart
+Navigator.push(context, new MaterialPageRoute(builder: (BuildContext context)
+ => new UsualNavscreen()));
+```
+
+### How do I use tab navigation and drawer navigation?
+
+In Material Design apps, there are two primary options for Flutter navigation:
+tabs and drawers. When there is insufficient space to support tabs, drawers
+provide a good alternative.
+
+
+#### Tab navigation
+
+In React Native, `createBottomTabNavigator` and `TabNavigation`  are used to
+show tabs and for tab navigation.
+
+<!-- skip -->
+```JavaScript
+// React Native
+import { createBottomTabNavigator } from 'react-navigation';
+
+const MyApp = TabNavigator(
+  { Home: { screen: HomeScreen }, Notifications: { screen: tabNavScreen } },
+  { tabBarOptions: { activeTintColor: "#e91e63" } }
+);
+```
+
+Flutter provides several specialized widgets for drawer and tab navigation:
+
+* [TabController](https://docs.flutter.io/flutter/material/TabController-class.html)—Coordinates the tab selection between a TabBar and a TabBarView.  
+
+* [TabBar](https://docs.flutter.io/flutter/material/TabBar-class.html) —Displays a horizontal row of tabs.  
+* [Tab](https://docs.flutter.io/flutter/material/Tab-class.html)—Creates a material design TabBar tab.  
+
```

Comment:
Delete. It makes the final list look disjointed.

Reply:
Done